src/components/WaitForOpponentStep.tsx

```diff
@@ -18,12 +22,10 @@ export default class WaitForOpponentStep extends React.PureComponent {
         </div>
         {selectedPlayId && (
           <div style={{ width: '100%' }}>
-            You&apos;ve chosen {PLAY_OPTIONS.find(option => option.id === selectedPlayId).name}
+            You&apos;ve chosen {Play[selectedPlayId]}
```

Comment:
I don't know how/if we should ensure that `selectedPlayId` is valid.

Reply:
I don't think we actually need to convert to and id and back. We should be able to iterate over the `Plays`, with the button for a given play triggering the action with that play baked in.